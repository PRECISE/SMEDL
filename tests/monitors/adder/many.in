{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [1],
    "aux": {
        "arbitrary": "json data"
     }
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [2],
    "aux": [1, 2, 3]
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [3.5],
    "aux": "foo"
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [4.0],
    "aux": 36
} 
{
    "fmt_version": [2, 0],
    "channel": "conn",
    "event": "measurement",
    "params": [5.0],
    "aux": {
    }
} 
