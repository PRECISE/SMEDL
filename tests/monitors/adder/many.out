{
    "fmt_version": [2, 0],
    "channel": "_Adder_sum",
    "event": "Adder.sum",
    "identities": [],
    "params": [1.0],
    "aux": {
        "arbitrary": "json data"
     }
} 
{
    "fmt_version": [2, 0],
    "channel": "_Adder_sum",
    "event": "Adder.sum",
    "identities": [],
    "params": [3.0],
    "aux": [1, 2, 3]
} 
{
    "fmt_version": [2, 0],
    "channel": "_Adder_sum",
    "event": "Adder.sum",
    "identities": [],
    "params": [6.5],
    "aux": "foo"
} 
{
    "fmt_version": [2, 0],
    "channel": "_Adder_sum",
    "event": "Adder.sum",
    "identities": [],
    "params": [10.5],
    "aux": 36
} 
{
    "fmt_version": [2, 0],
    "channel": "_Adder_sum",
    "event": "Adder.sum",
    "identities": [],
    "params": [15.5],
    "aux": {
    }
} 
